libft/sources/strings/strings_internal.h

```diff
@@ -15,36 +15,16 @@
 
 # include <stddef.h>
 # include <stdlib.h>
+# include <stdbool.h>
 # include "strings.h"
 # include "../memory/memory.h"
```

Comment:
You can just include libft.h for this, also some of the headers above can be included there as well